modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogManagerImpl.java

```diff
@@ -254,6 +270,11 @@ public int activeCatalogVersion(long timestamp) {
         return catalogAt(timestamp).version();
     }
 
+    @Override
+    public int latestCatalogVersion() {
+        return catalogByVer.lastEntry().getKey();
```

Comment:
What of `catalogByVer` map is empty?

Reply:
At the `CatalogManagerImpl#start`, we add a "zero version" and also check this in the `CatalogManagerSelfTest#testLatestCatalogVersion`.